### PR TITLE
a few simple improvements to tour.jl

### PR DESCRIPTION
**Pull request by jrevels**
- uniform multiline comments
- add note about how consumers/producers can impose the structure that works best for them
- make `groupby` example slightly more complex to demonstrate multi-column index structures
- move complex filter example to later in the tour (after basic indexing)

### CODE REVIEW

File: examples/tour.jl [hunk 1]
```diff
@@ -90,32 +88,36 @@ Onda.log("wrote out $path_to_annotations_file")
...
+# and annotations are both represented in flat tables, enabling you to easily
+# impose whatever indexing structure is most convenient for your use case.
+#
+# For example, if you wish to primarily access signals by recording and kind,
+# you can easily create a structure with that index via `groupby`:
 target = rand(signals.recording)
-grouped = groupby(signals, :recording)
-grouped[(; recording=target)]
+grouped = groupby(signals, Cols(:recording, :kind))
+grouped[(target, "eeg")]
```
Comment (hannahilea) [inline examples/tour.jl:120]: i did not know you could do this!

**Review by hannahilea — APPROVED** [review]
Cool!

Tiny feature creep request: example of updating all `file_path`s in a table, e.g., to convert from relative to absolute (which is enough of an example to demonstrate how S3Paths would be created, without needing to add an AWSS3 dependency to this example file).

**Comment by jrevels** [comment]
> example of updating all file_paths in a table, e.g., to convert from relative to absolute (which is enough of an example to demonstrate how S3Paths would be created, without needing to add an AWSS3 dependency to this example file).

hmmm well the generated paths here are absolute already I think, so that would be perhaps too contrived...maybe there's a better example? though I guess in general this question is really "how do I apply a function to a DataFrame column" which gets somewhat into fundamental "read DataFrames documentation" territory compared to things that Onda should be documenting...

for completeness' sake:
```jl
df.column = f.(df.column) # in-place, can also use `transform!`
transform(df, :column => ByRow(f) => :column) # out-of-place
```

Maybe could include a `Tables.rowmerge` example though, since that function's a bit of a hidden gem that is very useful in composition with Onda's API

**Comment by hannahilea** [comment]
> Maybe could include a Tables.rowmerge example though, since that function's a bit of a hidden gem that is very useful in composition with Onda's API

^ yeah, that'd be great.

**Comment by jrevels** [comment]
> ^ yeah, that'd be great.

sounds good! will do so in another PR

**Pull request merged**